### ISSUE TITLE
[CODE] port HF bloom to hivemind

### ISSUE THREAD

**Issue by justheuristic** [bigscience-workshop/petals#2]
__Why:__ so we can play with in in inference mode

Original bloom code: https://github.com/huggingface/transformers/pull/17474

The quest is to
- implement bloom __transformer layer__  as a hivemind expert.
- prepare a huggingface model that only has bloom embeddings and logits, but runs all transformer layers via hivemind.RemoteExpert
